lib/fun_ruby/modules/enum/max.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module FunRuby
+  module Modules
+    # @private
+    module Enum
+      # Returns the maximum element of the enumerable
+      #
+      # @since 0.1.0
+      #
+      # @param function [#call/1] ->(left, right) { returns Boolean }
```

Comment:
Are you sure that the function should return a boolean?

Reply:
Ouch, I forgot about it  <=> , thx